CenterForOpenScience/osf.io
# [ENG-2816] Deleting project of for-project registration does not delete node

## Purpose

Currently draft registrations with deleted nodes show up in the my registrations view, but they should not.

## Changes

- Clean-up queryset to exclude deleted nodes.

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify deleted nodes are removed from my registrations page.
- Verify

What are the areas of risk?

Any concerns/considerations/questions that development raised?

## Documentation



## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/browse/ENG-2816
